pool/src/lib.rs

```diff
@@ -892,6 +872,11 @@ impl LendingPool {
         assert_with_error!(env, flags.is_active, Error::NoActiveReserve);
     }
 
+    fn require_borrowing_enabled(env: &Env, reserve: &ReserveData) {
+        let flags = reserve.configuration.get_flags();
```

Comment:
Since `borrowing_enabled` is a `pub` field the call `.get_flags()` could be omitted